org/checkForSemanticFormatting.ts

```diff
@@ -0,0 +1,13 @@
+import { danger, warn, fail } from "danger"
+
+const semanticFormat = /^(fix|feat|build|chore|ci|docs|style|refactor|perf|test)(?:\(.+\))?:.+$/
```

Comment:
```suggestion
const semanticFormat = /^(fix|feat|build|chore|ci|docs|style|refactor|perf|test)!?(?:\(.+\))?:.+$/
```

I'd suggest adding an optional bang to the regex, to cover the case of a breaking change expressed via semantic convention. (See bullet #3 under https://www.conventionalcommits.org/en/v1.0.0/#summary)

Currently looks like such a message, while conforming to spec, would get failed here.